File: DotNet/ModelTracKer/Models/Competition.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace ModelTracKer.Models
{
    [Table("competition")]
    public class Competition
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public string Name { get; set; }

        public ICollection<opp_competition> oppCompetitions { get; set; }
    }
}

File: Java/tracker/tracker/src/main/java/com/tacker/tracker/models/Competition.java
package com.tacker.tracker.models;

import jakarta.persistence.*;
import lombok.*;

import java.util.Set;

@Entity
@Table(name = "competition")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class Competition {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int Id;

    private String value;

    @OneToMany(mappedBy = "competition")
    private Set<OppCompetition> oppCompetitionSet;

}
